tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -20,13 +20,15 @@
 use Sonata\AdminBundle\Exception\NoValueException;
 use Sonata\AdminBundle\Tests\Fixtures\Entity\FooToString;
 use Sonata\AdminBundle\Twig\Extension\SonataAdminExtension;
+use Sonata\AdminBundle\Twig\Extension\SonataTemplatesExtension;
```

Comment:
Please create a separate test class, and use prophecy

Reply:
This is done to avoid exceptions in these tests.

Reply:
Okay